sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -174,6 +194,28 @@ export interface PartitionContext {
     readonly partitionId: string;
 }
 
+// @public
+export interface PartitionOwnership {
```

Comment:
Update documentation here and for InMemoryPartitionManager as well

Reply:
Basically just review this markdown file and add all missing documentation

Reply:
Ohh. got it. I usually don't check markdown file.